test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/time/Constants.java

```diff
@@ -33,6 +33,7 @@
     }
 
     public class User {
-        public static final String YODA = "yoda";
+
+        public static final String YODA = "adminUser";
```

Comment:
Please rename the constant

Reply:
This change won't be part of the PR... will revert it now. I was hoping to fix a failing test by renaming this user.

Reply:
ok